tool/tctl/common/lock_command.go

```diff
@@ -47,14 +47,21 @@ func (c *LockCommand) Initialize(app *kingpin.Application, config *servicecfg.Co
 	c.mainCmd.Flag("user", "Name of a Teleport user to disable.").StringVar(&c.spec.Target.User)
 	c.mainCmd.Flag("role", "Name of a Teleport role to disable.").StringVar(&c.spec.Target.Role)
 	c.mainCmd.Flag("login", "Name of a local UNIX user to disable.").StringVar(&c.spec.Target.Login)
-	c.mainCmd.Flag("node", "UUID of a Teleport node to disable.").StringVar(&c.spec.Target.Node)
+	// Locking a node is now deprecated, but we still support it for backwards compatibility.
+	// Previously, locking a node would lock only the `ssh_service` from that node to
+	// access Teleport but didn't prevent any other roles that the same instance could run.
+	// Now, `tctl lock --server-id` should be used instead to lock the entire server
+	// and all roles that it runs (including the `ssh_service`) from accessing Teleport.
+	// TODO: DELETE IN 15.0.0
+	c.mainCmd.Flag("node", "UUID of a Teleport node to disable.").Hidden().StringVar(&c.spec.Target.Node)
```

Comment:
If `Node` is set we should display a deprecation warning, IMO.

Reply:
Added in 7a6bfee379265e0e0eebb2abc1c28b91c897e033